rotkehlchen/accounting/structures/base.py

```diff
@@ -359,10 +359,15 @@ def from_history_base_entry(cls, event: HistoryBaseEntry) -> 'StakingEvent':
 
     def serialize(self) -> dict[str, Any]:
         data = {
-            'event_type': self.event_type.serialize(),
             'asset': self.asset.identifier,
             'timestamp': self.timestamp,
             'location': str(self.location),
         }
+        if not (
```

Comment:
Would need a comment here why this is happening. Explan the endpoint only returns one type of event so this should not be serialized